steps.d.ts

```
@@ -1,17 +1,45 @@
 /* eslint-disable */
 /// <reference types='codeceptjs' />
-type steps_file = typeof import('./steps_file.js')
+type steps_file = () => {
+	grabDimensionsOfCurrentPage: () => Promise<{
+		width: number
+		height: number
+		deviceScaleFactor: number
+	}>
+	checkIfCurrentDeviceIsTablet: () => Promise<{
+		isDeviceTablet: Boolean
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)